src/models/customs-declaration-model.js

```diff
@@ -54,32 +56,39 @@ const getAuthorityByCheckCode = (checkCode) => {
   return (checkCode?.length) ? checkCodeToAuthorityMapping[checkCode.toUpperCase()] : ''
 }
 
-const getCustomsDeclarationStatus = (commodities) => {
-  if (commodities?.length) {
-    if (commodities.some(c => c.checks.some(chk => isErrorDecisionCode(chk.decisionCode)))) {
+const getCustomsDeclarationStatus = (items, finalisation) => {
```

Comment:
the items parameter rename can be undone and left as commodities.
This is what it will be in the new `Imports Data API`

Reply:
Does it not make sense to have it consistently named as items now and then flip to commodities once the new data schema is finalised and implemented?

Reply:
Okay, let's go with aligning the names to the current response model for now.
This is all subject to change anyway